docs/extending.md

```diff
@@ -132,4 +132,40 @@ bash
 # continue from previous shell
 (venv_gandlf) $> coverage run -m pytest --device cuda; coverage report -m
 ```
+## Logging
+
+### Use loggers instead of print
+We use the native `logging` [library](https://docs.python.org/3/library/logging.html) for logs management.
+It is already configured. So, If you are extending the code, please use loggers instead of prints.
```

Comment:
```suggestion
We use the native `logging` [library](https://docs.python.org/3/library/logging.html) for logs management. This gets automatically configured when GaNDLF gets launched. So, if you are extending the code, please use loggers instead of prints.
```